### PR TITLE
fix: courses path links

### LINKED ISSUE #47: Fix "Choose what you want to learn" section links

### PR DESCRIPTION
**Pull request by Quiddlee**
## What type of PR is this? (select all that apply)

- [ ] 🍕 Feature
- [x] 🐛 Bug Fix
- [ ] 🚧 Breaking Change
- [ ] 🧑‍💻 Code Refactor
- [ ] 📝 Documentation Update

## Related Tickets & Documents

- Related Issue #47 
- Closes #47

## Screenshots, Recordings

image

## Added/updated tests?

- [ ] 👌 Yes
- [x] 🙅‍♂️ No, because they aren't needed
- [ ] 🙋‍♂️ No, because I need help

## [optional] What gif best describes this PR or how it makes you feel?
BrexbyBrexbyMonkeyGIF

### CODE REVIEW

**Review by natanchik — COMMENTED** [review]
Could you please replace links with RS School links and remove words 'upturn' and 'invert' from titles?

**Comment by Quiddlee** [comment]
> Could you please replace links with RS School links and remove words 'upturn' and 'invert' from titles?

Done

File: src/app/services/data/courses-path.data.ts [hunk 1]
```diff
@@ -46,8 +48,12 @@ export const coursesPath: CoursesPath[] = [
...
       {
-        linkTitle: 'React invert',
-        href: 'https://rs.school/react/'
+        linkTitle: 'React',
+        href: buildUrl('/courses/reactjs')
+      },
+      {
+        linkTitle: 'Angular',
+        href: buildUrl('/courses/angular')
       }
```
Comment (natanchik) [inline src/app/services/data/courses-path.data.ts:57]: It's cool that you added 👍

File: src/app/services/data/courses-path.data.ts [hunk 2]
```diff
@@ -5,6 +5,7 @@ import NodeJSIcon from '@/assets/icons/nodejs.webp';
 import AWSFundamentalsIcon from '@/assets/icons/aws-fundamentals.webp';
 import AWSDeveloperIcon from '@/assets/icons/aws-developer.webp';
 import { type CoursesPath } from './courses-data.types';
+import { buildUrl } from '@/app/services/platform';
```
Comment (natanchik) [inline src/app/services/data/courses-path.data.ts:8]: 👍

File: src/app/services/data/courses-path.data.ts [hunk 3]
```diff
@@ -29,12 +30,13 @@ export const coursesPath: CoursesPath[] = [
...
+        href: buildUrl('/courses/javascript-mentoring-program')
       },
       {
-        linkTitle: 'JS/TS/FE Fundamentals (EN) invert',
-        href: 'https://rs.school/js-en/'
+        linkTitle: 'JS/TS/FE Fundamentals (EN)',
+        href: buildUrl('/courses/javascript-mentoring-program')
       }
     ]
   },
```
Comment (natanchik) [inline src/app/services/data/courses-path.data.ts:42]: We'll have to remember to update the link if we add a second page.

**Review by natanchik — APPROVED** [review]
I think everything is OK 👍.

**Review by andron13 — APPROVED** [review]

**Comment by dzmitry-varabei** [comment]
@Quiddlee thanks!

**Pull request merged**